packages/faustwp-core/src/config/index.ts

```diff
@@ -33,11 +34,23 @@ export function setConfig(_config: FaustConfig) {
   return once(() => {
     config = _config;
 
-    const { experimentalPlugins: plugins } = _config;
+    const { experimentalPlugins, plugins } = _config;
+    // combine both sets of plugins until experimentalPlugins is fully deprecated
+    const allSupportedPlugins = [
+      ...(experimentalPlugins || []),
+      ...(plugins || []),
+    ];
 
-    plugins?.forEach((plugin) => {
+    allSupportedPlugins?.forEach((plugin) => {
       plugin?.apply?.(hooks);
     });
+
+    if (experimentalPlugins?.length) {
+      // log to cli if experimentalPlugins is used since it's being deprecated
+      console.warn(
```

Comment:
We may want to use our `warnLog` util here so it looks consistent with the rest of our logs and is colored appropriately.